source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -62,11 +62,11 @@ There are a couple of different ways to specify when SSL/TLS is required:
 
   .. NOTE:: With both :dovecot_core:ref:`ssl=yes <ssl>` and :dovecot_core:ref:`ssl=required <ssl>` it's still possible that the client attempts to do a cleartext authentication before enabling SSL/TLS, which exposes the cleartext password to the internet.
 
-             Dovecot attempts to indicate this to the IMAP clients via the LOGINDISABLED capability, but many clients still ignore it and send the password anyway. There is unfortunately no way for Dovecot to prevent this behavior. The POP3 standard doesn't have an equivalent capability at all, so the POP3 clients can't even know if the server would accept a cleartext authentication.
+      Dovecot attempts to indicate this to the IMAP clients via the LOGINDISABLED capability, but many clients still ignore it and send the password anyway. There is unfortunately no way for Dovecot to prevent this behavior. The POP3 standard doesn't have an equivalent capability at all, so the POP3 clients can't even know if the server would accept a cleartext authentication.
 
 * The main difference between :dovecot_core:ref:`ssl=required <ssl>` and :dovecot_core:ref:`auth_allow_cleartext=no <auth_allow_cleartext>` is that if :dovecot_core:ref:`ssl=required <ssl>`, it guarantees that the entire connection is protected against eavesdropping (SSL/TLS encrypts the rest of the connection), while :dovecot_core:ref:`auth_allow_cleartext=no <auth_allow_cleartext>` only guarantees that the password is protected against eavesdropping (SASL mechanism is encrypted, but no SSL/TLS is necessarily used). Nowadays you most likely should be using SSL/TLS anyway for the entire connection, since the cost of SSL/TLS is cheap enough. Using both SSL/TLS and non-cleartext authentication would be the ideal situation since it protects the cleartext password even against man-in-the-middle attacks.
 
-  .. NOTE:: The cleartext authentication mechanisms are always allowed (and SSL not required) for connections from localhost, as they're assumed to be secure anyway. This applies to all connections where the local and the remote IP addresses are equal. IP ranges specified by :dovecot_core:ref:`login_trusted_networks` setting are assumed to be secure. If you want localhost to be trusted, it needs to be included in this setting. It is only considered secure automatically, not trusted.
+  .. NOTE:: Since v3.0.0;v2.4.0 With ssl=required, the cleartext authentication mechanisms are no longer allowed from trusted connections, such as localhost.
```

Comment:
"secured" != "trusted". The change affects only trusted connections, not secured (= localhost) connections. Also should use `.. changedsince::` tag, right?

Reply:
it's in a note, so probably does not work.

Reply:
secured != trusted is still wrong.

Reply:
So I'd leave the original few sentences and then add new text:

```
The cleartext authentication mechanisms are always allowed (and SSL not required) for connections
from localhost, as they're assumed to be secure anyway. This applies to all connections where the
local and the remote IP addresses are equal. Since v3.0.0;v2.4.0 With ssl=required, the cleartext
authentication mechanisms are no longer allowed from IP addresses in
:dovecot_core:ref:`login_trusted_networks`.
```

Reply:
You also need to update login_trusted_networks setting's documentation.